src/components/ExpiryIndicator/ExpiryIndicator.tsx

```diff
@@ -8,14 +8,16 @@ import {
   Strong,
   StyledPill,
   StyledTooltip,
+  Text,
 } from "./ExpiryIndicator.styles";
 import { getExpiryTranslation } from "./helpers";
 
 type ExpiryIndicatorProps = {
   expiry: Date;
+  className?: string;
 };
 
-const ExpiryIndicator: FC<ExpiryIndicatorProps> = ({ expiry }) => {
+const ExpiryIndicator: FC<ExpiryIndicatorProps> = ({ expiry, className }) => {
   const { t } = useTranslation();
 
   const hasPassed = useMemo(
```

Comment:
```hasExpired``` could be more clear

Reply:
Expiry is a `Date`. It's used in other types like `UnsignedOrder` and `Order`, so it's best to keep things consistent.

Reply:
```hasPassed``` -> ```hasExpired``` was what i meant, my bad.

Reply:
Fixed!